Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -834,4 +834,29 @@ class Testball < Formula
     desc_cache.unlink
     formula_file.unlink
   end
+
+  def test_style
+    foo_file = CoreTap.new.formula_dir/"foo.rb"
+    foo_file.write <<-EOS.undent
+      class Foo < Formula
+        url 'https://example.com/foo-1.0'
+      end
+    EOS
+
+    rubocop_yml = HOMEBREW_LIBRARY/".rubocop.yml"
+    rubocop_yml.write <<-EOS.undent
+      Style/Documentation:
+        Enabled: false
+
+      Style/StringLiterals:
+        EnforcedStyle: double_quotes
+    EOS
+
+    assert_match "Prefer double-quoted strings", cmd_fail("style")
+    assert_match "1 file inspected, 1 offense detected", cmd_fail("style", "#{foo_file}")
```

Comment:
Don't worry! It's always a bit sad to let go of a PR, but it would be strange if every single one of them was a success story. On the positive side, I think we all learned a few things here. :)